recipe/meta.yaml

```
@@ -23,12 +24,13 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python {{ python_min }}
+    - setuptools >=69.0
     - pip
   run:
+    - python >={{ python_min }}
     - alpha-shapes >=1.1.0
-    - python >=3.8
-    - setuptools
+    - setuptools >=69.0
     - h5py >=2.10
     - numpy >=1.20
```

This is not in the metadata and it is an easy fix upstream. You will need to import the rankwarning from the new excpetion name espace in numpy 2.0. For now, pinning it to <2 would do the trick.

```suggestion
    - numpy >=1.20,<2
```